docs/rst/installation/linux_installation.rst

```diff
@@ -2,3 +2,316 @@
+In Ubuntu, it can be installed running:
+
+.. code-block:: bash
+
+    sudo apt install libxerces-c-dev
+
+GTest (optional)
+""""""""""""""""
+
+GTest is a unit testing library for C++.
+By default, *eProsima Fast DDS QoS Profiles Manager* does not compile tests.
+It is possible to activate them with the opportune CMake configuration options when calling colcon or CMake.
+
+GTest installation can be performed either following the `GTest installation instructions <https://google.github.io/googletest/>`_, or including the `Gtest repository <https://github.com/google/googletest>`_ in the workspace directory, running:
+
+.. code-block:: bash
+
+    git clone https://github.com/google/googletest ~/fastdds_qosprofman_ws/src/googletest-distribution
+
+.. note::
+
+    If the colcon deployment is followed, and GTest has been installed in the workspace directory, the GTest dependency needs to be included in the *Library* ``colcon.pkg`` file.
+    Please check the :ref:`colcon build procedure <installation_build_source_linux>` for further information.
+
+CLI tool
+^^^^^^^^
+
+The *eProsima Fast DDS QoS Profiles Manager CLI* depends on the *Library tool* and also on the following package.
+
+Docopt
+""""""
+
+`docopt <http://docopt.org/>`_ is a CLI description language that defines the interface of the command line applications and generates a parser for it automatically.
+In Ubuntu, the dependency can be installed running:
+
+.. code-block:: bash
+
+    sudo apt install libdocopt-dev
+
+GUI tool
+^^^^^^^^
+
+.. warning::
+
+    This tool is still in development process.
+
+Documentation tool
+^^^^^^^^^^^^^^^^^^
+
+The *eProsima Fast DDS QoS Profiles Manager Documentation* depends on the *Library tool* and also on the following packages.
+
+Doxygen
+"""""""
+
+`Doxygen <https://www.doxygen.nl/>`_ is a C++ documentation generator.
+It is required to build the :ref:`Library API reference <fastdds_qos_profiles_manager_lib_api_reference>`.
+In Ubuntu, the dependency can be installed running:
+
+.. code-block:: bash
+
+    sudo apt install doxygen
+
+Python3 virtual environment (recommended)
+"""""""""""""""""""""""""""""""""""""""""
+
+It is recommended to install the *Documentation* tool dependencies in a `python3 <https://www.python.org/downloads/>`_ virtual environment.
+That would avoid polluting the user's installation setup.
+
+Deploy the project in a python3 virtual environment by running:
+
+    .. code-block:: bash
+
+        # Python3 virtual environment installation
+        sudo apt install python3.10-venv
+
+        # Virtual environment deployment
+        cd <path_to_virtual_environment_directory>
+        python3 -m venv fastdds_qosprofman_venv
+        source fastdds_qosprofman_venv/bin/activate
+
+    .. note::
+
+        As long as colcon performs a recursive search to build packages inside the workspace directory, it is recommended to deploy the virtual environment outside of the workspace directory in order to avoid conflicts.
```

Comment:
```suggestion
    .. code-block:: bash

        # Python3 virtual environment installation
        sudo apt install python3.10-venv

        # Virtual environment deployment
        cd <path_to_virtual_environment_directory>
        python3 -m venv fastdds_qosprofman_venv
        source fastdds_qosprofman_venv/bin/activate

    .. note::

        As colcon performs a recursive search to build packages inside the workspace directory, it is recommended to deploy the virtual environment outside of the workspace directory in order to avoid conflicts.
```